GTFS.DB.SQLite/Collections/SQLiteShapesCollection.cs

```diff
@@ -191,6 +191,34 @@ public bool Remove(string entityId)
             }
         }
 
+        /// <summary>
+        /// Removes a range of entities by their IDs
+        /// </summary>
+        /// <param name="entityId"></param>
+        /// <returns></returns>
+        public void RemoveRange(IEnumerable<string> entityIds)
```

Comment:
I'm a little uncertain as to whether RemoveRange is the best name, since these are more like a DeleteByIds... but that name is also not great so not sure what you think

Reply:
well you started it in StopCollection with commit ebebefcdcd94e801dcb2e6f44e62e32367221f6f, I just copied the code

Reply:
I agree that the naming is wrong though. RemoveRange is for removing a given number of entities from a given index from a list.

Reply:
I guess let's leave it for now until we come up with a better name

Reply:
well I do need to complete this PR soon though, I have another Capture PR that's waiting on this one